test/LndClient/RPCSpec.hs

```diff
@@ -108,9 +83,9 @@ btcClient :: IO Client
 btcClient = do
   env <- btcEnv
   let user = btcRpcUser env
-  let pass = btcRpcPassword env
+  let passw = btcRpcPassword env
   let url = btcRpcUrl env
-  getClient url user pass
+  getClient (unpack url) user passw
```

Comment:
is this the only place where `url` is used?
it can be String instead of Text to avoid unpack computation
not big deal anyway